portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,13 +19,24 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
You can remove the TODO comment here now.

Reply:
Done